repos/fdbt-site/tests/pages/__snapshots__/searchOperators.test.tsx.snap

```diff
@@ -58,107 +63,133 @@ exports[`pages searchOperator should render a selection error when the user trie
-              className="govuk-fieldset__legend"
+            <h2
+              className="govuk-body-l govuk-!-margin-bottom-0"
+              id="operator-search-results"
             >
-              <h2
-                className="govuk-body-l govuk-!-margin-bottom-0"
-                id="operator-search-results"
-              >
-                Your search for '
-                <strong>
-                  blac
-                </strong>
-                ' returned
-                <strong>
-                   
-                  1
-                   result
-                </strong>
-              </h2>
-            </legend>
-            <FormElementWrapper
-              errorClass=""
-              errorId="add-operator-checkbox-0"
-              errors={Array []}
+              Your search for '
+              <strong>
+                blac
+              </strong>
+              ' returned
+              <strong>
+                 
+                1
+                 result
+              </strong>
+            </h2>
+          </legend>
+          <FormElementWrapper
+            errorClass=""
+            errorId="add-operator-0"
+            errors={Array []}
+          >
+            <div
+              className="govuk-checkboxes"
             >
+              <p
+                className="govuk-hint"
+                id="traveline-hint-text"
+              >
+                Select the operators results and click add operator(s). This data is taken from Bus Open Data Service (BODS).
+              </p>
+              <p
+                className="govuk-hint"
+                id="noc-hint-text"
+              >
+                You will see that all operator names are followed by a series of characters. These characters are the operator's National Operator Code (NOC).
+              </p>
               <div
-                className="govuk-checkboxes"
+                className="govuk-checkboxes__item"
+                key="checkbox-item-Blackpool"
               >
-                <p
-                  className="govuk-hint"
-                  id="traveline-hint-text"
-                >
-                  Select the operators results and click add operator(s). This data is taken from the Traveline National Dataset.
-                </p>
-                <p
-                  className="govuk-hint"
-                  id="noc-hint-text"
-                >
-                  You will see that all operator names are followed by a series of characters. These characters are the operator's National Operator Code (NOC).
-                </p>
-                <div
-                  className="govuk-checkboxes__item"
-                  key="checkbox-item-Blackpool"
+                <label
+                  className="govuk-label govuk-checkboxes__label"
+                  htmlFor="add-operator-checkbox-0"
+                  id="operator-to-add-0"
+                  onClick={[Function]}
+                  role="input"
```

Comment:
Do we need a role on an input?

Reply:
type script will complain about adding click to none interactive elements